File: mongodb/python/delete_one.py
from pymongo import MongoClient
from bson import ObjectId
from db import client
import pprint
 
# get to sample_mflix
dbname=client['course']

# get collection
collection=dbname['billings']

# document id to find 
document_to_find={"_id": ObjectId("573a1390f29313caabcd42e8")}


# find document
result=collection.delete_one(document_to_find)


# print result
pprint.pprint(result)

File: mongodb/nodejs/delete_one.js
// import mongo client
const {client} = require('./db');


const main = async ()=> {
    try {
        //open connection with mongo
        await client.connect();
        console.log('Connection successfully to ATLAS');
        // get db
        const dbName = client.db('course');
        // get ventas collection
        const collection = dbName.collection('billings');
        // create query
        const query = { code_billing: { $eq: 4 }};
        // insert document
        let result = await collection.deleteOne(query);
        console.log(result)
    } catch (error) {
        console.log("**__** ~ main ~ error:", error);        
    } finally {
        // close connection
        await client.close()
    }
}

main();